packages/data/src/api/tokenById/tokenById.ts

```diff
@@ -0,0 +1,47 @@
+import { fetchGraphQl } from "../../graphql/fetch";
+import { errorContractAddress, errorToken } from "./tokenById.mock";
+import { tokenByIdQuery } from "./tokenbyId.query";
+
+import { TokenByIdResults } from "./tokenById.types";
+
+export const checkParams = (tokenId: string, contractAddress: string) => {
+  if (typeof tokenId !== "string") {
+    return errorToken;
```

Comment:
Don't think we want to import anything from mocks into the implementation.  Not only is it confusing, I think this would include the big mock object in any bundle using this method

Reply:
ok,  will fix this by tomorrow